lib/cisco_node_utils/cmd_ref/bgp.yaml

```diff
@@ -109,14 +109,14 @@ confederation_id:
     config_set_append: '<state> confederation identifier <id>'
 
 confederation_peers:
-  # TODO: Supported in XR but broken as identifier is nvgened on a seperate line.
   default_value: ""
```

Comment:
Got it. Note  Rob - I can move the default to the common area of the yaml block instead of doing it 2x.

confederation_id:
    default_value: []
  cli_ios_xr:
    # Note: Only applies to global BGP; not VRF applicable
    config_get_token_append: '/^bgp confederation identifier (\d+|\d+.\d+)$/'
    config_set_append: ' <state> bgp confederation identifier <id>'
  cli_nexus:
    config_get_token_append: '/^confederation identifier (\d+|\d+.\d+)$/'
    config_set_append: '<state> confederation identifier <id>'